test_data/powershell_transcript_ger.txt

```diff
@@ -0,0 +1,56 @@
+﻿**********************
+nStart der Windows PowerShell-Aufzeichnung

```

Comment:
is this leading `n` part of the log file?

Reply:
I'll retest this on one of my Windows Machines and report back later.

Reply:
thanks